### ISSUE TITLE
Customizing text at top (beyond Parent Title)

### ISSUE THREAD

**Issue by mbtrillions** [GitMurf/roam-highlighter#114]
There are times when I am pulling highlights when I want to add a series of tags and metadata to the top of my highlights (while I can copy it in within roam, it would be nice to be able to save this as sometimes I want to repeat the structure for multiple articles I am highlighting).  I tried simply adding tags in the "Parent Title for Highlights" box, but seems like it may not handle line breaks?

**Comment by GitMurf** [comment]
@mbtrillions did you make sure to click the "Add" button and give it a name (see GIF below)? If you do you can always switch to that tag structure in the future. It will be available for any website at any time. Even when you close and re-open your browser.

![0eWDVfYGxH](https://user-images.githubusercontent.com/64155612/98454655-f8fb8980-211b-11eb-8e2b-0c638bc9106b.gif)

**Comment by GitMurf** [comment]
I am getting the impression though that this may not be your question/issue though? Could you maybe explain a little more what you are trying to do and take a screenshot showing me what it looks like? Thanks!

**Comment by mbtrillions** [comment]
I was indeed trying to create a new "Parent Title" format, but it wasn't
showing up when I actually tried to highlight the page I was working on.
Now when I tried again, it worked! I added multiple line breaks, added them
with and without sub-bullets, and it all worked out.  So perhaps I
inadvertently did something else that activated a bug or some other issue,
but I was not able to replicate my error :)
